sycl/unittests/CMakeLists.txt

```diff
@@ -33,4 +33,5 @@ add_subdirectory(program_manager)
 add_subdirectory(assert)
 add_subdirectory(Extensions)
 add_subdirectory(windows)
+add_subdirectory(event)
```

Comment:
That's an interesting change in this late commit...

Reply:
I tried to resolve merge conflict. This line has already added to CMakeLists.txt